password_validator.rb

```diff
@@ -3,5 +3,17 @@
 # - include at least one of these characters: !, @, $, %, &
 
 def valid?(password)
-  
+  if password.length >= 8 && password.include?('!')
+    return true
+  elsif password.length >= 8 && password.include?('@')
+    return true
+  elsif password.length >= 8 && password.include?('$')
+    return true
+  elsif password.length >= 8 && password.include?('%')
+    return true
+  elsif password.length >= 8 && password.include?('&')
+    return true
+  else
+    return false
+end
```

Comment:
Don't forget to indent this penultimate 'end' so it won't get deleted by mistake

Reply:
Very good point - thank you